src/App.tsx

```diff
@@ -1,57 +1,28 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
-import { peopleFromServer } from './data/people';
+import { DropDownMenu } from './components/DropDownMenu/DropDownMenu';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selected, setSelected] = useState<Person | null>(null);
```

Comment:
```suggestion
  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
```

wil be more informative